roguedex-dev/roguedex
# Update Chrome manifest to v3

### What changed
Updating Chrome manifest from v2 to v3, as per the instructions [here](https://developer.chrome.com/docs/extensions/develop/migrate) and [here](https://stackoverflow.com/questions/63308160/how-to-migrate-manifest-version-2-to-v3-for-chrome-extension).

Fixes #12 

### How to test
* Load the extension unpacked in Chrome dev mode `chrome://extensions`
    * You may need to rename `manifest_chrome.json` to `manifest.json`
* Check that the extension isn't throwing any errors in the dashboard 
* Check that the extension works as expected on `https://pokerogue.net/`
